x-pack/plugins/security_solution/public/detection_engine/rule_management_ui/components/rules_table/upgrade_prebuilt_rules_table/upgrade_prebuilt_rules_tour.tsx

```diff
@@ -0,0 +1,131 @@
+  EuiText,
+  EuiTourStep,
+  useEuiTour,
+} from '@elastic/eui';
+import { noop } from 'lodash';
+import type { FC } from 'react';
+import React, { useEffect, useMemo } from 'react';
+import { NEW_FEATURES_TOUR_STORAGE_KEYS } from '../../../../../../common/constants';
+import { useKibana } from '../../../../../common/lib/kibana';
+import { useIsElementMounted } from '../rules_table/guided_onboarding/use_is_element_mounted';
+import * as i18n from './translations';
+
+export interface RulesFeatureTourContextType {
+  steps: EuiTourStepProps[];
+  actions: EuiTourActions;
+}
+
+export const PER_FIELD_UPGRADE_TOUR_ANCHOR = 'perFieldUpgradeTour';
+
+const TOUR_STORAGE_KEY = NEW_FEATURES_TOUR_STORAGE_KEYS.RULE_MANAGEMENT_PAGE;
+const TOUR_POPOVER_WIDTH = 400;
+
+const tourConfig: EuiTourState = {
+  currentTourStep: 1,
+  isTourActive: true,
+  tourPopoverWidth: TOUR_POPOVER_WIDTH,
+  tourSubtitle: '',
+};
+
+const stepsConfig: EuiStatelessTourStep[] = [
+  {
+    step: 1,
+    title: i18n.UPDATE_TOUR_TITLE,
+    content: <EuiText>{i18n.UPDATE_TOUR_DESCRIPTION}</EuiText>,
+    stepsTotal: 1,
+    children: <></>,
+    onFinish: noop,
+    maxWidth: TOUR_POPOVER_WIDTH,
+  },
+];
+
+export const PrebuiltRulesUpgradeTour: FC = () => {
+  const { storage } = useKibana().services;
+
+  const restoredState = useMemo<EuiTourState>(
+    () => ({
+      ...tourConfig,
+      ...storage.get(TOUR_STORAGE_KEY),
+    }),
+    [storage]
+  );
+
+  const [tourSteps, tourActions, tourState] = useEuiTour(stepsConfig, restoredState);
+
+  useEffect(() => {
+    const { isTourActive, currentTourStep } = tourState;
+    storage.set(TOUR_STORAGE_KEY, { isTourActive, currentTourStep });
+  }, [tourState, storage]);
+
+  const shouldShowRuleUpgradeTour = useIsElementMounted(PER_FIELD_UPGRADE_TOUR_ANCHOR);
+
+  const enhancedSteps = useMemo(
+    () =>
+      tourSteps.map((item, index) => ({
+        ...item,
+        content: (
+          <>
+            {item.content}
+            {tourSteps.length > 1 && (
+              <>
+                <EuiSpacer size="s" />
+                <EuiFlexGroup responsive={false} gutterSize="s" alignItems="center">
+                  <EuiFlexItem grow={false}>
+                    <EuiButtonIcon
+                      iconType="arrowLeft"
+                      aria-label={'previous'}
+                      display="empty"
+                      disabled={index === 0}
+                      onClick={tourActions.decrementStep}
+                    />
+                  </EuiFlexItem>
+                  <EuiFlexItem grow={false}>
+                    <EuiButtonIcon
+                      iconType="arrowRight"
+                      aria-label={'next'}
+                      display="base"
+                      disabled={index === tourSteps.length - 1}
+                      onClick={tourActions.incrementStep}
+                    />
+                  </EuiFlexItem>
+                </EuiFlexGroup>
+              </>
+            )}
```

Comment:
Since the tour only has one step, you don't need all of this code. This can be simply:

```ts
  const enhancedSteps = useMemo(
    () =>
      tourSteps.map((item) => ({
        ...item,
        content: item.content,
      })),
    [tourSteps]
  );
```